web_src/css/base.css

```diff
@@ -2784,3 +2784,7 @@ table th[data-sortt-desc] .svg {
 .color-text-light-2 {
   color: var(--color-text-light-2);
 }
+
+.menu-scrolling-rss-button {
```

Comment:
IIRC there is already a `.ui.right` in `base.css`, so we do not need it.
```
.ui.right:not(.action) {
  float: right;
}
```

Reply:
ah good catch, I'll update